rosidl_generator_cpp/CMakeLists.txt

```diff
@@ -53,10 +53,6 @@ if(AMENT_ENABLE_TESTING)
     SKIP_INSTALL
   )
 
-  if(NOT WIN32)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
-  endif()
```

Comment:
This is necessary to build the test with C++11 and all warning enabled. It could be moved down to the test target though.